projects/qcms/Dockerfile

```diff
@@ -15,7 +15,7 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-builder
-RUN apt-get update && apt-get install -y mercurial
+RUN apt-get update && apt-get install -y mercurial make autoconf automake libtool curl cmake python llvm-dev libclang-dev clang
```

Comment:
Do you need 
"make curl cmake python llvm-dev libclang-dev clang"
All of these should be on the image and we should be using the latest clang already provided on image.

Reply:
I just copied the stuff from `serde_json/Dockerfile` as an example. It's probably not needed. I'll take it out.